docs/_scripts/prepare_notebooks_for_ci.py

```diff
@@ -70,10 +69,13 @@ def is_comment(code: str) -> bool:
     return code.strip().startswith("#")
 
 
-def has_blocklisted_command(code: str) -> bool:
+def has_blocklisted_command(code: str, metadata: dict) -> bool:
+    if 'hide_from_vcr' in metadata:
+        return True
+    
     code = code.strip()
-    for blocklisted_command in BLOCKLIST_COMMANDS:
-        if blocklisted_command in code:
+    for blocklisted_pattern in BLOCKLIST_COMMANDS:
```

Comment:
if you have time, would be great to change the other blocklisted cells to use this, but can be a separate PR

Reply:
gonna merge this PR because there are a lot of cells to do this for and idk how to do this smartly (i.e. not manually)